modules/worker_thread/worker_thread.c

```diff
@@ -157,7 +157,7 @@ bool worker_thread_publisher_task_publish_from_ISR(struct worker_thread_publishe
     chSysLockFromISR();
     struct worker_thread_publisher_msg_s* msg = chPoolAllocI(&task->pool);
     chSysUnlockFromISR();
-
+    memset(msg, 0, sizeof(struct worker_thread_publisher_msg_s));
```

Comment:
This needs to move down a few lines.

Reply:
I've corrected a bug in my PR where msg->size was not initialized. Please pull that patch in. This memset should not be necessary.